app/manifest/v2/_base.json

```diff
@@ -71,6 +71,16 @@
     "activeTab",
     "webRequest",
     "*://*.eth/",
+    "*://*.crypto/",
+    "*://*.x/",
+    "*://*.nft/",
+    "*://*.dao/",
+    "*://*.bitcoin/",
+    "*://*.klever/",
+    "*://*.wallet/",
+    "*://*.blockchain/",
+    "*://*.888/",
+    "*://*.zil/",
```

Comment:
We'll need to look into what changes we might need to make to manifest v3 as well

Reply:
yeah, weren't sure what was needed in manifest v3

Reply:
Doesn't look like the ENS ipfs resolution needed to be in manifest v3 but we'll need to defer to you @brad-decker if this is something we need to add.